### PR TITLE
[nt] Add calculations for total cells and calculate percentages

### PR DESCRIPTION
**Pull request by nathaniel-mage**
# Summary

### Show percentage of invalid and missing values in Metrics table
- Adds a new parameter to the parse (needs column types to get number of columns)
- Handles edge case of 0 rows or cols, but 0 rows is more likely.

# Tests

Launched on Localhost 

### Blank dataset
image

### Normal dataset
image

cc:

@johnson-mage

### CODE REVIEW

**Comment by nathaniel-mage** [comment]
Note that if you'd like for these metrics to also be shown with a progress bar you can update the constants file to include `  'total_invalid_value_count',
  'total_null_value_count'` inside `PERCENTAGE_KEYS`.

File: mage_ai/frontend/components/datasets/overview/utils.ts [hunk 1]
```diff
@@ -7,14 +7,17 @@ import {
...
   WARN_KEYS,
 } from '../constants';
 
-export function createMetricsSample(statistics) {
+export function createMetricsSample(statistics, colTypes) {
   const stats = Object.keys(statistics);
+  const types = Object.values(colTypes);
   const metricRows = Array(METRICS_KEYS.length).fill(0);
-
+  const total_cells = (statistics?.count === 0 || types?.length === 0)
```
Comment (johnson-mage) [inline mage_ai/frontend/components/datasets/overview/utils.ts:19]: camelCase

**Review by johnson-mage — APPROVED** [review]
check comment

File: mage_ai/frontend/components/datasets/overview/utils.ts (hunk 1 above)
Comment (nathaniel-mage) [reply mage_ai/frontend/components/datasets/overview/utils.ts:19]: ah good catch will fix.
